### DISCUSSION TITLE
Where do I define the port I want an instance to use?

### DISCUSSION THREAD

**Discussion by myndxero** [vladmandic/sdnext#191] [Q&A]
I cannot use my auto1111 webui-user.bat here so I'm not sure where to look to change the port.  I scoured settings and was able to set 96% of my usual command line arguments I use in auto's, but port wasn't one of them. Using the set commandline_args caused the launch to fail.

Thanks so much!

PS - I really hope this fork gains significant traction. Really like what's going on here and just weary of Auto's absenteeism. Also, if unaware, I believe Dr. Furkan Gözükara of the SEC courses and channel on youtube has taken an interest in this fork.

EDIT:

Caught this when paying more attention to the console, just no where I see to edit this in particular. I will keep looking. I run multiple web UIs for different things at times so defining different ports is useful to me.
![image](https://user-images.githubusercontent.com/113237671/232425799-1b36f208-cc3b-4243-b3ff-0df3071487f6.png)

**Comment by vladmandic** [comment]
`--port`

on how to set params, just create a new batch file, for example `my-webui.bat` with a single line:

> @webui.bat --port 8080
